src/commands/extensionRelease.js

```diff
@@ -58,6 +58,8 @@ async function releaseExtension() {
         log.title('Release a TAO extension');
 
         await release.loadConfig();
+        await release.initialiseGithubClient();
+        await release.verifyCredentials();
```

Comment:
Isn't this too early? If you try to initialise the Github client here, it doesn't necessarily know the repo. The TAO instance and repo are selected in `selectTarget()`.

When I ran `taoRelease extensionRelease` I got the error:
```
Error: TypeError: Cannot read property 'getRepoName' of null
```

Reply:
Interesting. The errors I got locally were happening when I put it above the loadConfig, besides that I wasn't getting any other error... was this a fresh install or something? Can you paste the whole log of of the flow?